registry/Scripts/script.sh

```diff
@@ -205,6 +211,11 @@ echo fetching user credentials
 HTPASSWD_DIR="/root/auth"
 mkdir -p $HTPASSWD_DIR
 fetchCredentials
+returnCode=$?
```

Comment:
@jadarsie Updated the code a bit here again as non zero return code was not failing the deployment. Above changes will address both cases. Let me know if you feel otherwise.

Reply:
sorry, I missed it, it should be `exit $ERR_MISSING_USER_CREDENTIALS` in line 133